### ISSUE TITLE
markdownToDraft crashes on empty list items

### ISSUE THREAD

**Issue by joepio** [Rosey/markdown-draft-js#20]
Similar to #17 

````
Draft.EditorState.createWithContent(Draft.convertFromRaw(markdownToDraft('-')))
````

````
Draft.EditorState.createWithContent(Draft.convertFromRaw(markdownToDraft('1.')))
````

**Comment by Rosey** [comment]
Darn, thanks :(

**Comment by Rosey** [comment]
Fix should be out now as 0.3.3
